onnxscript/function_libs/torch_aten/param_manipulation.py

```diff
@@ -55,4 +94,18 @@ def separate_input_attributes_from_arguments(
         - A list of ONNX inputs.
         - An ordered dictionary of ONNX attribute names and values.
     """
-    raise NotImplementedError()
+    # args, kwargs and param_schemas should be all in order
+    # user might not specify all attributes
+    assert len(args) + len(kwargs) <= len(param_schemas)
```

Comment:
What do we do when len(args) > len(param_schemas) or when kwargs contains unknown arguments?